src/main/java/com/daml/extensions/testing/ledger/DamlScriptRunner.java

```diff
@@ -91,22 +93,25 @@ public DamlScriptRunner build() {
     }
 
     private ProcessBuilder command() {
-      String sandboxHost = "localhost";
-
-      return new ProcessBuilder()
+        String sandboxHost = "localhost";
+        List<String> commands = new ArrayList<>();
+        commands.add(damlCommand());
+        commands.add("script");
+        for (Path relativeDarPath : darPath) {
+            commands.add("--dar");
+            commands.add(relativeDarPath.toString());
+        }
```

Comment:
Apparently this fails, `daml script` does not support multiple `--dar` options. However, it is not needed for a script. A DAR should contain all what is necessary for a script.

Reply:
How should we start the script then? Give the script dar as the first or as a separate parameter?

Reply:
The current `DamlScriptRunner` is capable of running of running arbitrary scripts. What I'd do is remove `daml.yaml` related code from it, and that's it. One can use this as is.